web/app/view/UserDialogController.js

```diff
@@ -48,19 +44,5 @@ Ext.define('Traccar.view.UserDialogController', {
             });
         }
         button.up('window').close();
-    },
-
-    showAttributesView: function (button) {
```

Comment:
It was the same as in `DeviceDialogController`.
I moved it to `BaseEditDialogWithAttributesController` actually git recognized it like renaming from `DeviceDialogController`.